app/views/spree/products/index.xml.builder

```diff
@@ -0,0 +1,26 @@
+xml.instruct!
```

Comment:
So I started off thinking, "Ugh this is full duplication" but then I realized that we _might_ have to change the XML to accomodate Google, and then it would not be duplication so I'm :+1: about it now.

Reply:
i kept fighting the with the other builder file trying to re-use it to build `xml` and `rss` based on the request format, but couldn't find a way.  but i agree, at least having a separate `xml` response, we can update as needed to keep pace with any changes google rolls out and not have to worry about the `rss`